include/dca/parallel/mpi_concurrency/mpi_concurrency.hpp

```diff
@@ -68,6 +73,12 @@ class MPIConcurrency final : public virtual MPIInitializer,
     return util::getBounds(id(), number_of_processors(), dmn);
   }
 
+  // Using gather with no gang uses the entire concurrency.
```

Comment:
Doc quote probably needs to move to mpi_gather.hpp.  The rest looks like it should be deleted.